src/app/advancedsearch/advancedsearch.component.spec.ts

```diff
@@ -50,7 +51,8 @@ describe('AdvancedsearchComponent', () => {
         FooterNavbarComponent,
         AboutComponent,
         ContactComponent,
-        ModalComponent
+        ModalComponent,
+        TopbarComponent
```

Comment:
one ad-hoc way to know which tests are depending on the present change or new component is run tests on each change and see which tests are failing.